symfony/plugins/orangehrmLeavePlugin/test/Service/BasicWorkScheduleTest.php

```diff
@@ -0,0 +1,127 @@
+        $startDate = new DateTime('2021-12-25 09:00');
+        $endDate = new DateTime('2021-12-25 17:00');
+        $workShift->setStartTime($startDate);
+        $workShift->setEndTime($endDate);
+        $employeeWorkShift->setWorkShift($workShift);
+        $employeeDao = $this->getMockBuilder(EmployeeDao::class)
+            ->onlyMethods(['getEmployeeWorkShift'])
+            ->getMock();
+
+        $employeeDao->expects($this->exactly(1))
+            ->method('getEmployeeWorkShift')
+            ->with(1)
+            ->willReturn($employeeWorkShift);
+
+        $employeeService = $this->getMockBuilder(EmployeeService::class)
+            ->onlyMethods(['getEmployeeDao'])
+            ->getMock();
+
+        $employeeService->expects($this->exactly(1))
+            ->method('getEmployeeDao')
+            ->willReturn($employeeDao);
+
+        $basicWorkSchedule = $this->getMockBuilder(BasicWorkSchedule::class)
+            ->onlyMethods(['getEmployeeService'])
+            ->getMock();
+
+        $basicWorkSchedule->expects($this->exactly(1))
+            ->method('getEmployeeService')
+            ->willReturn($employeeService);
+        $basicWorkSchedule->setEmpNumber(1);
+
+        $expectResult = new WorkShiftStartAndEndTime($startDate, $endDate);
+
+        $this->assertEquals($expectResult, $basicWorkSchedule->getWorkShiftStartEndTime());
+    }
+
+    public function testGetWorkShiftStartEndTimeEmployeeWorkShiftEmpty()
+    {
+        $employeeDao = $this->getMockBuilder(EmployeeDao::class)
+            ->onlyMethods(['getEmployeeWorkShift'])
+            ->getMock();
+
+        $employeeDao->expects($this->exactly(1))
+            ->method('getEmployeeWorkShift')
+            ->with(1)
+            ->willReturn(null);
+
+
+        $employeeService = $this->getMockBuilder(EmployeeService::class)
+            ->onlyMethods(['getEmployeeDao'])
+            ->getMock();
+
+        $employeeService->expects($this->exactly(1))
+            ->method('getEmployeeDao')
+            ->willReturn($employeeDao);
+
+
+        $startDate = new DateTime('2021-12-25 09:00');
+        $endDate = new DateTime('2021-12-25 17:00');
+        $defaultWorkShift = new WorkShiftStartAndEndTime($startDate, $endDate);
+
+        $workShiftService = $this->getMockBuilder(WorkShiftService::class)
+            ->onlyMethods(['getWorkShiftDefaultStartAndEndTime'])
+            ->getMock();
+
+        $workShiftService->expects($this->exactly(1))
+            ->method('getWorkShiftDefaultStartAndEndTime')
+            ->willReturn($defaultWorkShift);
+
+        $basicWorkSchedule = $this->getMockBuilder(BasicWorkSchedule::class)
+            ->onlyMethods(['getEmployeeService', 'getWorkShiftService'])
+            ->getMock();
+
+        $basicWorkSchedule->expects($this->exactly(1))
+            ->method('getEmployeeService')
+            ->willReturn($employeeService);
+        $basicWorkSchedule->expects($this->exactly(1))
```

Comment:
`$this->exactly(1)` ===> `$this->once()`

Reply:
Changed all places